store/store.go

```diff
@@ -0,0 +1,212 @@
+package store
+
+//import "log"
+import "time"
+import "golang.org/x/crypto/blake2b"
+import "sync"
+import "fmt"
+
+type NodeID [32]byte
+type Key NodeID
+
+type item struct {
+	value     string
+	expire    time.Time
+	republish time.Time
+	origPub   NodeID
+}
+
+type localItem struct {
+	value     string
+	repubTime time.Time
+}
+
+type items struct {
```

Comment:
Maybe name this one to `remoteItems` to match `localItems`?

Reply:
Makes sense. Done.